Improve community standards

JackGilmore opened the pull request:
Relates to https://github.com/OpenDataScotland/the_od_bods/issues/103

Adds some global files to include the following community documents in our repositories:

- Issue templates: bug reports and feature requests
- Pull request template
- Code of conduct

JackGilmore committed:
Update issue templates

JackGilmore committed:
Reintroduce code of conduct and add issue templates

JackGilmore committed:
PR and issue templates

KarenJewell reviewed (commented):
### Code of Conduct
The slack join link is not appropriate for reporting guideline violations - unless we want to specify a channel or persons. Otherwise remove and restrict to email and twitter.

### Pull Request Template
Maybe this is the place to mention using black? Or should we link our own style guide?
Maybe add an option for "I need help with creating a test?" something like that

JackGilmore committed:
PR feedback changes

JackGilmore commented:
Committed some changes in https://github.com/OpenDataScotland/.github/commit/2d948b38191814f87098e8acd5ba73c8f5edec91

### Code of Conduct

Fair point. I think I had just copied our contact us info from the site. Agree that Twitter and email should be fine. Have removed Slack.

### Pull Request Template

Referring to Black and style guide is probably best done in the contributing file. I've added a wee spiel in there for now but ultimately we may not need to ask contributors to run Black locally if we can run it automatically with GitHub actions as per #118.

Added the following line to the template to direct for support with testing
```
<!-- If you are unsure how to add tests, please ask for guidance when you open your PR -->
```
